.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "ansible/roles/nginx"]
```

Comment:
Question : why git submodules and not a galaxy requirements?

(http://docs.ansible.com/ansible/galaxy.html#installing-multiple-roles-from-a-file)